qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -141,13 +190,48 @@ def run(
                     * none: The job is not shared at any level.
 
                 If the job share level is not specified, the job is not shared at any level.
-            job_tags: Tags to be assigned to the jobs. The tags can subsequently be used
+            job_tags: Tags to be assigned to the job. The tags can subsequently be used
                 as a filter in the :meth:`jobs()` function call.
             validate_qobj: If ``True``, run JSON schema validation against the
-                submitted payload
+                submitted payload. Only applicable if a Qobj is passed in.
```

Comment:
We might want to deprecate this. The schema's in terra are deprecated now so if we want to continue to offer this as an option we'll have to port over the schema compilation and validation and use the ibmq schemas repo.

Reply:
Yes I opened #800 to deprecate it. Just haven't gotten to it yet.